WebApplication/ClientApp/src/reducers/uiFlagsReducer.js

```diff
@@ -140,7 +140,7 @@ export default function(state = initialState, action) {
       case uiFlagsActionTypes.REJECT_PARAMETERS_EDITED_MESSAGE:
          return { ...state, parametersEditedMessageRejected: action.show };
       case uiFlagsActionTypes.SHOW_MODAL_PROGRESS:
-         return { ...state, modalProgressShowing: action.visible};
+         return { ...state, modalProgressShowing: action.visible, stats: null };
```

Comment:
If state contains `stats` then a modal dialog shows it.  The "async" processings were doing `stats` reset after SignalR communication gets started.  Fix is to reset it before any UI is shown.

I joined "show dialog" and "reset stats" into reducers. (not sure if it's a proper React way)